library/src/window_impl.cxx

```diff
@@ -247,6 +247,8 @@ void window_impl::UpdateDynamicOptions()
   this->Internals->Renderer->ShowFilename(this->Internals->Options.getAsBool("ui.filename"));
   this->Internals->Renderer->SetFilenameInfo(
     this->Internals->Options.getAsString("ui.filename-info"));
+  this->Internals->Renderer->SetAnimationnameInfo(
+    this->Internals->Options.getAsString("ui.animationname-info"));
```

Comment:
I'm not sure what this is doing ? displaying the name of the animation in the filename actor ?

Reply:
<img width="570" alt="Screenshot 2024-01-24 at 10 14 26 AM" src="https://github.com/f3d-app/f3d/assets/2395780/0366189f-c064-4147-bda3-8693fe1e8dfb">

Yes, here is a screenshot. It shows the current animation name when cycling through. Basically, I copied all of the `ui.filename-info` to create a new one for animation names `ui.animationname-info`.

Reply:
Interesting! I'm not sure we want to show that information here though. I think the cheatsheet should be fine. Wdyt @Meakk ?

Reply:
Lets not keep this.